### ISSUE TITLE
Metamask integration

### ISSUE THREAD

**Issue by bogdynator** [xclud/web3dart#40]
This library has any implementation regarding Metamask?

**Comment by edTheGuy00** [comment]
[A previous version was working](https://github.com/simolus3/web3dart/tree/master/example/metamask), not sure if this one since I see the metamask example got removed.

**Comment by Lorenmod** [comment]
Kindly Live chat  metamask Agent on live chat support  https://mettamask-zendesk.com/
if you have issues or concerns

**Comment by TheGreatAxios** [comment]
Currently, I believe there are better libraries for Metamask integration specifically.

**Comment by rennylangat** [comment]
Yes with the current library you can interact with Metamask. You will to use WalletConnect package as well

**Comment by rennylangat** [comment]
An update to this thread. The current WalletConnect packages have not been completely moved to V2. Moreover V1 does not work entirely so if you happen to have a way out please create an issue or discussion.
